GetIntoTeachingApi/Models/GetIntoTeaching/TeachingEventSearchRequest.cs

```diff
@@ -11,7 +11,7 @@ public class TeachingEventSearchRequest : ICloneable
         [SwaggerSchema("Set to filter results to a radius (in miles) around the postcode.")]
         public int? Radius { get; set; }
         [SwaggerSchema("Set to filter results to a type of teaching event. Must match an `typeId` of the `TeachingEvent` schema.")]
-        public int? TypeId { get; set; }
```

Comment:
This will need to stay in for backwards-compatibility initially; we can remove it once we've updated the clients to use `TypeIds`